frontend/javascripts/admin/dataset/dataset_upload_view.tsx

```diff
@@ -650,17 +668,8 @@ class DatasetUploadView extends React.Component<PropsWithFormAndRouter, State> {
                   name="initialTeams"
                   label="Teams allowed to access this dataset"
                   hasFeedback
-                  rules={[
-                    {
-                      required: !isDatasetManagerOrAdmin,
-                      // @ts-expect-error ts-migrate(2322) FIXME: Type 'string | null' is not assignable to type 'st... Remove this comment to see the full error message
-                      message: !isDatasetManagerOrAdmin
-                        ? messages["dataset.import.required.initialTeam"]
-                        : null,
-                    },
-                  ]}
                 >
-                  <Tooltip title="Except for administrators and dataset managers, only members of the teams defined here will be able to view this dataset.">
+                  <Tooltip title="The dataset can be seen by administrators, dataset managers and by teams that have access to the folder to which the dataset is uploaded. If you want to grant additional teams access, define these teams here.">
```

Comment:
I cannot trigger this tooltip any longer. Maybe it should be triggered by an info icon as the tooltip next to the "Target Folder" form item.